<launch>
	<node name="get_map_server" pkg="autonomous_exploration" type="get_map_server">
		<param name="LETHAL_OBSTACLE" type="int" value= "80" />
	</node>

<!--
	<param name="robot_radius" type="double" value="$(optenv ROS_ROBOT_RADIUS 0.18)" />


	<param name="laser_max_range" type="double" value="$(optenv ROS_LASER_MAX_RANGE 3.0)" />


	<param name="min_distance" type="double" value="$(optenv ROS_MIN_DISTANCE 1.0)" />

	<param name="lethal_cost" type="int" value="$(optenv ROS_LETHAL_COST 65)" />

	<param name="gain_threshold" type="double" value="$(optenv ROS_GAIN_THRESHOLD 35.0)" />
	<param name="gain_change" type="double" value="$(optenv ROS_GAIN_CHANGE 1.5)" />

	<param name="min_gain_threshold" type="double" value="$(optenv ROS_MIN_GAIN_THRESHOLD 10.0)" />

	<param name="map_path" type="string" value="$(env HOME)/maps/"/>
-->
	<node name="explore_client" pkg="autonomous_exploration" type="explore_client" output="screen" />
	<!-- <node name="explore_server" pkg="autonomous_exploration" type="explore_server" output="screen" /> -->
</launch>
